lib_comfyui/webui_settings.py

```diff
@@ -22,8 +22,21 @@ def get_additional_argv():
     return [arg.strip() for arg in shared.opts.data.get('comfyui_additional_args', '').split()]
 
 
+def get_url():
+    global html_object_url
+    return html_object_url if html_object_url else f'http://127.0.0.1:{get_port()}'
+
+
 def get_port():
     webui_argv = get_additional_argv()
     port_index = webui_argv.index('--port') if '--port' in webui_argv else -1
     settings_port = webui_argv[port_index + 1] if 0 <= port_index < len(webui_argv) - 1 else None
```

Comment:
can we change `if 0 <= port_index < len(webui_argv) - 1` to `port_index in range(len(webui_argv))`?